### DISCUSSION TITLE
Make engineer talent "Co-Author" usable in singleplayer

### DISCUSSION THREAD

**Discussion by marpul358** [FakeFishGames/Barotrauma#8968] [Ideas and suggestions]
Make engineer talent "Co-Author" usable in singleplayer. Currently talent description says:

_You can co-write a book with a Mechanic. To do this, select their character and have them select yours. (Open eachothers health menus)_

This does not seem to work in singleplayer. Would be nice if it would just require something like "ragdoll your engineer near a mechanic" or "as an engineer slap your mechanic with a stun baton".

**Comment by Kaillera** [comment]
Have your mechanic or engineer assigned the wait command. Use that mechanic or engineer and grab onto your counterpart. Control the counterpart and grab them back.

**Comment by marpul358** [comment]
I stand corrected. Just tried it and it works in singleplayer. The key thing was the wait command.

**Comment by James-E-A** [comment]
Alternative suggestion: don't require bots to reciprocate the Health interface; simply holding an engineer bot with the talent who is _idle_ (lounging icon) for 10 seconds should be enough

**Reply by AshEdwardsDesign** [reply]
+1 would be nice if this was streamlined for SP.
